docs/jest-integration.md

```diff
@@ -0,0 +1,312 @@
+# Fuzzing using the Jest integration
+
+To use fuzzing in your normal development workflow a tight integration into the
+[Jest test framework](https://jestjs.io/) is provided. Using this, fuzz tests
+can be executed alongside your normal unit tests and seamlessly detect problems
+on your local machine or in your CI, and check that found bugs stay resolved
+forever.
+
+Furthermore, the Jest integration enable great IDE support, so that individual
+inputs can be run, or even debugged, similar to what you would expect from
+normal Jest tests.
+
+The Jest integration provides two modes of execution, which will be explained in
+detail further down on this page.
+
+- **Fuzzing Mode**: Fuzzing a function through a Jest test.
+- **Regression Mode**: Using initially provided seeds and inputs of found
+  problems to execute the Jest test with.
+
+## Setting up the Jazzer.js Jest integration
+
+Jest enables the execution of tests by third party tools through custom test
+runners. Jazzer.js provides such a runner in the `@jazzer.js/jest-runner`
+package.
+
+To use the integration add a `dev-dependency` to `@jazzer.js/jest-runner` to
+your project.
+
+```shell
+npm install --save-dev @jazzer.js/jest-integration
```

Comment:
See the other remark on getting Jazzer.js as a dep

Reply:
done